djstripe/models/base.py

```diff
@@ -95,8 +96,10 @@ def api_retrieve(self, api_key=None, stripe_account=""):
             field
             for field in self._meta.get_fields(include_parents=True)
             if field.is_relation and field.one_to_many
-            # TODO: Avoiding circular import problems. Is there a better way?
-            and field.related_model._meta.model_name == "account"
+            # Avoid circular import problems by using the app registry to
+            # get the model class rather than a direct import.
+            and field.related_model
+            is apps.get_model(app_label="djstripe", model_name="account")
```

Comment:
Is an identity check (`is`) correct here?

Reply:
I can't think if a reason why it wouldn't be, unless the same class can somehow end up with different identities? The alternative is:
```
and field.related_model._meta.app_label == "djstripe"
and field.related_model._meta.modle_name == "account"
```

Reply:
Yeah, I think you're right.